scripts/main-yield-charts.main.v1.0.0.js

```diff
@@ -0,0 +1,257 @@
+}, []);
+const getRentArrayWith = async ({year= '2023',month='01',risk=1,portfolio, typeUniverse}) => {
+  const url = `https://91o7sqo7s3.execute-api.us-east-1.amazonaws.com/prod/${portfolio}Yields`
+  var myHeaders = new Headers();
+  myHeaders.append("Content-Type", "application/json");
+  var raw = JSON.stringify({"year": year,"month": month,"risk": risk, typeUniverse});
+  var requestOptions = {
+    method: 'POST',
+    headers: myHeaders,
+    body: raw,
+    redirect: 'follow'
+  };
+  return fetch(url, requestOptions).then(response => response.json()).then(mapAsDataArray)
+};
+const mapExtractingPercentages = (dataArray) => dataArray.map(data => ([data[0], (data[1] - 1) * 100]))
+
+const startingPoint = ["", 0];
+
+const mapExtractingPercentagesWithTooltipInfo = (dataArray) => dataArray.map((data, index)=>([	
+data[0],
+(data[1] - 1) * 100,
+`Rentabilidad acumulada: ${numberToPercent(data[1])} \n
+  Rentabilidad mensual: ${numberToPercent((data[1] / (dataArray[index - 1]?.[1] ?? 1 )))}`,
+]))
+const getYield = (dataArrays) => (yieldType) => (portfolioType) => ({
+  annual: numberToPercent(dataArrays[portfolioType]['1a'].at(-1)[1]),
+  accumulated: numberToPercent(dataArrays[portfolioType].max.at(-1)[1]),
+  last: numberToPercent(dataArrays[portfolioType].max.at(-1)[1] / dataArrays[portfolioType].max.at(-2)[1])
+  })[yieldType];
+
+const sleep = m => new Promise(r => setTimeout(r, m))
+
+const assignToElementIfPossible = (elementId, value) => {
+  if (document.getElementById(elementId)?.textContent) {
+    document.getElementById(elementId).textContent = value;
+  }
+}
+function updateBottomYieldDetails(dataArrays) {
+  assignToElementIfPossible('wallet-accum-yield', getYield(dataArrays)('accumulated')('wallet'));
+  assignToElementIfPossible('wallet-last-yield', getYield(dataArrays)('last')('wallet'));
+  assignToElementIfPossible('wallet-annual-yield', getYield(dataArrays)('annual')('wallet'));
+
+  assignToElementIfPossible('smart-conservative-accum-yield', getYield(dataArrays)('accumulated')('smartConservative'));
+  assignToElementIfPossible('smart-conservative-last-yield', getYield(dataArrays)('last')('smartConservative'));
+  assignToElementIfPossible('smart-conservative-annual-yield', getYield(dataArrays)('annual')('smartConservative'));
+  assignToElementIfPossible('smart-moderate-accum-yield', getYield(dataArrays)('accumulated')('smartModerate'));
+  assignToElementIfPossible('smart-moderate-last-yield', getYield(dataArrays)('last')('smartModerate'));
+  assignToElementIfPossible('smart-moderate-annual-yield', getYield(dataArrays)('annual')('smartModerate'));
+  assignToElementIfPossible('smart-risky-accum-yield', getYield(dataArrays)('accumulated')('smartRisky'));
+  assignToElementIfPossible('smart-risky-annual-yield', getYield(dataArrays)('annual')('smartRisky'));
+  assignToElementIfPossible('smart-risky-last-yield', getYield(dataArrays)('last')('smartRisky'));
```

Comment:
Si haces un (tal vez hasta te cabía en webflow jaja)
```
const updateYieldsFor = type => {
  assignToElementIfPossible(`${type}-accum-yield`, getYield(dataArrays)('accumulated')(type));
  assignToElementIfPossible(`${type}-last-yield`, getYield(dataArrays)('last')(type));
  assignToElementIfPossible(`${type}-annual-yield`, getYield(dataArrays)('annual')(type));
}
```

Y acá ahorras harto
```suggestion
function updateBottomYieldDetails(dataArrays) {
  updateYieldsFor('wallet');
  updateYieldsFor('smartConservative');
  updateYieldsFor('smartModerate');
  updateYieldsFor('smartRisky');
```

Habría que ajustar los `pascal/kebab-case` si fuese necesario, no cacho ahí la implementación